marimo/_config/config.py

```diff
@@ -107,7 +105,7 @@ class ServerConfig(TypedDict, total=False):
         with Python's webbrowser module (eg, `"firefox"` or `"chrome"`)
     """
 
-    browser: Literal["default"] | str
+    browser: Union[Literal["default"], str]
```

Comment:
@mscolnick FYI now that we are using these classes in Pydantic, we need to use `Union` instead of the `|` operator for Python 3.8/3.9 compatibility.